app/src/main/java/org/hackillinois/android/view/MentorAdapter.kt

```diff
@@ -0,0 +1,43 @@
+package org.hackillinois.android.view
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import kotlinx.android.synthetic.main.mentor_recycleview.view.*
+import org.hackillinois.android.R
+import org.hackillinois.android.model.mentor.MentorModel
+
+class MentorAdapter internal constructor(private val mentorList: ArrayList<MentorModel>) : RecyclerView.Adapter<MentorAdapter.ViewHolder>() {
+
+    private lateinit var context: Context
+
+    inner class ViewHolder(parent: View) : RecyclerView.ViewHolder(parent)
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ViewHolder {
+        val view = LayoutInflater.from(parent.context).inflate(R.layout.mentor_recycleview, parent, false)
+        val viewHolder = ViewHolder(view)
+        context = parent.context
+        return viewHolder
+    }
+
+    override fun onBindViewHolder(holder: MentorAdapter.ViewHolder, position: Int) {
+        val mentor = mentorList[position]
+
+        holder.itemView.apply {
+            mentorCardLayout.setOnClickListener { view ->
+                /* val context = view.context
+                val intent = Intent(context, EventInfoActivity::class.java)
+                intent.putExtra("event_name", event.name)
+                context.startActivity(intent)*/
+            }
+
+            mentorName.text = mentor.name
+            mentorNumber.text = "#" + mentor.number
```

Comment:
You can do something like this in Kotlin: "#${mentor.number}" and it will create a string for you.